server/controllers/cookiePolicyController.test.ts

```diff
@@ -0,0 +1,81 @@
+import type { DeepMocked } from '@golevelup/ts-jest'
```

Comment:
Giving up for now at `Refactor middleware to controller` since it looks like things might be refactored and my brain is fried!